sdcm/cluster_aws.py

```diff
@@ -322,6 +322,9 @@ def _wait_public_ip(self):
             self._instance.reload()
 
     def restart(self):
+        result = self.remoter.run('curl http://169.254.169.254/latest/meta-data/instance-type -s')
```

Comment:
The ip is used by https://github.com/scylladb/scylla-ami/blob/master/ds2_configure.py#L96